photutils/isophote/isophote.py

```diff
@@ -716,7 +716,7 @@ def b4_err(self):
         """
         return self._collect_as_array('b4_err')
 
-    def to_table(self):
+    def to_table(self, key_properties=['main']):
```

Comment:
```suggestion
    def to_table(self, columns='main'):
```

Reply:
The `photutils.segmentation` module has this same idea of specifying the output table columns (https://github.com/astropy/photutils/blob/master/photutils/segmentation/properties.py#L470).  I'd like to keep the keyword name here consistent with that.

Also, mutable default keywords arguments are not recommended in python (although it may not lead to the typically undesired effects here).  But I think it's better to just have `'main'` and `'all'` be allowed as string inputs here.  Requiring a list with these single strings is a bit unintuitive.